comfyui_custom_scripts/extensions/webuiRequests.js

```diff
@@ -73,14 +73,15 @@ function onElementDomIdRegistered(callback) {
 }
 
 function patchUiEnv(thisClientId) {
-    if(thisClientId !== 'comfyui_general_tab') {
+    if(thisClientId !== 'sandbox_tab') {
```

Comment:
I know all the ids have comfyui in their names, but doing this confuses me more than anything. In my view, the variable should contain the full name in order to make things clear that it represents directly the HTML id property. What do you think?